tests/phpunit/includes/WP_Object_Cache_dummy.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class WP_Object_Cache_dummy {
```

Comment:
The filename needs to be `wp-object-cache-dummy.php` and the classname needs to be `WP_Object_Cache_Dummy`.